### ISSUE TITLE
Error. To solve this problem?

### ISSUE THREAD

**Issue by Sanchozuy** [petyasoft/Blum#3]
![Screenshot_1](https://github.com/petyasoft/Blum/assets/165782472/11e2266e-6bfc-42e9-ad95-678c87b004e4)

18:25:38.76 | ERROR | main | Thread 1 | 0, message='Attempt to decode JSON with unexpected mimetype: text/html', url=URL('https://gateway.blum.codes/v1/friends/balance')

**Comment by mesteruh** [comment]
There are problems on blum service

**Comment by Sanchozuy** [comment]
What should I do? Stupidly waiting?

**Comment by petyasoft** [comment]
yes, i fix it in next update, but fix will only affect the fact that i disable the output this error

**Comment by Maryoux** [comment]
It happens because the request to server didn't return with the expected result. So it's just fine, since the script will just try to request it again.
